src/python/pants/build_graph/mirrored_target_option_mixin.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from abc import abstractmethod, abstractproperty
+
+from future.utils import text_type
+
+from pants.util.memo import memoized_property
+from pants.util.meta import AbstractClass
+from pants.util.objects import datatype
+
+
+class MirroredTargetOptionDeclaration(AbstractClass):
```

Comment:
With only one implementation, this interface isn't necessary anymore.

Reply:
Removed!